hive-metastore/src/test/java/org/apache/iceberg/hive/legacy/TestHiveExpressions.java

```diff
@@ -36,7 +36,6 @@
 import static org.apache.iceberg.expressions.Expressions.notNull;
 import static org.apache.iceberg.expressions.Expressions.or;
 import static org.apache.iceberg.hive.legacy.HiveExpressions.simplifyPartitionFilter;
-import static org.apache.iceberg.hive.legacy.HiveExpressions.toPartitionFilterString;
 
 
 public class TestHiveExpressions {
```

Comment:
can we add tests for DATE and TIMESTAMP (and other datatypes we should support)?